tests/conftest.py

```diff
@@ -57,3 +61,26 @@ def dvc(repo_dir, git):
         yield dvc
     finally:
         dvc.scm.git.close()
+
+
+here = os.path.abspath(os.path.dirname(__file__))
+
+user = "user"
+key_path = os.path.join(here, "{0}.key".format(user))
+
+
+@pytest.yield_fixture()
```

Comment:
Hi @ag613915cao , I think @Suor was referring to using this decorator instead of just `@pytest.fixture`. See the example in this file, a few lines above.

Reply:
@shcheklein Yes. I know about that. What I mean here is the way reviewer provide feedback to contributor(s). I have at least 5 years contributing to open source projects. I can realize that.

For this point: I will update this within today. Thank you for your consideration.